assets/javascripts/discourse-assign/initializers/extend-for-assigns.js

```diff
@@ -658,24 +657,6 @@ function initialize(api) {
     }
   });
 
-  api.addUserMenuGlyph((widget) => {
```

Comment:
We should add a `.discourse-compatibility` entry so that an older version of the plugin is used on Discourse 3.1.0.beta3 or below.

Reply:
There seems to be one already present from this recent commit: https://github.com/discourse/discourse-assign/pull/458